src/Symfony/Component/Security/Acl/Permission/PermissionMapInterface.php

```diff
@@ -39,4 +39,11 @@ public function getMasks($permission, $object);
      * @return bool
      */
     public function contains($permission);
+
+    /**
+     * Returns a new instance of the MaskBuilder used in the permissionMap
+     *
+     * @return MaskBuilderInterface
+     */
+    public function getMaskBuilder();
```

Comment:
I just provided a PR for 2.7 and backward compatibility. I created anoter interface for this method.

I will maintain this PR according to the new one : #14001